loader/schemas.py

```diff
@@ -0,0 +1,232 @@
+
+# These schemas are based on the DOS schemas with minor modifications. See
+# https://github.com/ga4gh/data-object-service-schemas/blob/master/openapi/data_object_service.swagger.yaml#L594
+
+data_object_schema = {
+    'type': 'object',
+    'required': ['id', 'size', 'created', 'checksums', 'urls'],
```

Comment:
The description comments of "REQUIRED" or "OPTIONAL" should match what is currently required for the loader, as identified here in the `required` statement.
The values `size` and `checksums` are not actually used by the loader today, although I think it is reasonable/good to keep them as required in the schema as they may potentially be used by the loader in the future.